MAINTAINERS.md

```diff
@@ -0,0 +1,12 @@
+# Maintainers
+
+The following table lists the Kubescape project core maintainers:
```

Comment:
This file is just the regolibrary maintainers, no?

Reply:
ah, I see what you mean... yes it's only for rego - let me fix that